working_dir/10_Parameter_File/WebParameters.yml

```diff
@@ -0,0 +1,9 @@
+default:
```

Comment:
shouldn't this be `WebParameters_2dii.yml` and `WebParameters_docker.yml` ?
Also, I think at the current stage of the code this should have an entry for `template_location`

@Clare2D

Reply:
forget this comment, I did not realise this file is in `/working_dir/10_parameter_files`  this should not interfere with aything